src/pymorize/cli.py

```diff
@@ -286,6 +308,7 @@ def populate_cache(files: List, verbose, quiet, logfile, profile_mem):
 ################################################################################
 
 cli.add_command(ssh_tunnel_cli, name="ssh-tunnel")
+cli.add_command(scripts)
```

Comment:
It seems that this is not used, can it be cleaned up, or finalized?

Reply:
as above